Design/Muon/ADS_data_structure.md

```diff
@@ -0,0 +1,285 @@
+# Sample Logs
+
+We want to use the sample logs of the workspaces to hold information about how these workspaces were produced. This means we don't need to carry around information about the groups and pairs, and we can always reliably and quickly discern this information from the logs.
+
+In the current GUI, the only way to get this information is to reverse-engineer the workspace name. This is a bad idea, but can easily be fixed. Even worse, there is some information we simply cannot obtain this way (for example, which detector id's were combined to give a particular group? Was a time offset applied?). This means the workspace and its name alone cannot be used to figure out what was done to calculate it.
+
+### Proposed sample log structure for every muon workspace
+
+We create a new group in the NeXuS style called "MuonAnalysis" and store all the parameters in there. There is a rough split between "pre-process" parameters and "group" or "pair" parameters; roughly following the earlier proposal for the sub-algorithms of `MuonProcess`;
```

Comment:
Ah, ok. I had hoped there was some sort of nesting structure available in the logs. In which case, we can simply amend the names I have given, so for example `analysis_crop_x_min` instead of `crop_x_min`. Then all relevant logs can be found by selecting those which start with `analysis`, or some similar string.